modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/display/context/DDLViewRecordsDisplayContext.java

```diff
@@ -98,7 +99,17 @@ public DDLViewRecordsDisplayContext(
 
 		_formDDMTemplateId = formDDMTemplateId;
 
-		_user = (User)_liferayPortletRequest.getAttribute(WebKeys.USER);
+		User user = (User)_liferayPortletRequest.getAttribute(WebKeys.USER);
```

Comment:
Hey @linolaoj Can you use the `PortalUtil.getUser` instead?

Reply:
pushed changes